members.txt

```diff
@@ -20,5 +20,5 @@ undergraduate researchers
 visitors and collaborators
  - Jigyasa Nigam 
  - Leonard and Matilda
-
+ - Lisa Je
```

Comment:
Can you put yourself alphabetically? We can leave L&M at the end

Reply:
Done and committed again